_layouts/docwithnav.html

```diff
@@ -1,15 +1,31 @@
 {% include dochead.html %}
-<!-- Search box for Google Custom Search Engine -->
-<div id="gcse-wrapper">
-	<gcse:search></gcse:search>
-</div>
-<!-- Navigation menu -->
-<div id="wrapper">
-	{% include nav_v1.0.html %}
-</div>
-<!-- Content body -->
-<div class="bodywrapper">
-	{% include documentation.html %}
+<div class="container-fluid">
+	<div class="row">
+		<div class="col-xs-1 hidden-md hidden-lg navbuttonarrow">
+		<!-- Nav menu button for smaller (mobile) views-->
+			<a href="javascript:;" class="togglenav">
+				<img src="{{site.url}}/img/mobile/chevron-right.png" style="display:inline-block"/>
+				<img src="{{site.url}}/img/mobile/chevron-left.png" style="display:none"/>
+			</a>
+		</div>
+		<div class="gcse-wrapper" id="gcse-searchbox">
+			<!-- Search box for Google Custom Search Engine -->
+			<gcse:searchbox></gcse:searchbox>
+		</div>
+	</div>
+	<div class="row">
+		<!-- Navigation menu -->
+		<div class="col-md-3 hidden-sm hidden-xs" id="wrapper">
+			{% include nav_v1.0.html %}
+		</div>
+		<!-- Content body -->
+		<div class="col-md-8 col-sm-11 col-xs-11 bodywrapper">
+			{% include documentation.html %}
+		</div>
+	</div>
+	<div class="gcse-wrapper">
```

Comment:
I would keep this as a top-level container. It displays floating over everything else, so there's no reason to nest it